opm/simulators/flow/GenericOutputBlackoilModule.cpp

```diff
@@ -325,6 +325,13 @@ outputInjLog(std::size_t reportStepNum)
     this->logOutput_.injection(reportStepNum);
 }
 
+template<class FluidSystem>
+void GenericOutputBlackoilModule<FluidSystem>::
+calc_initial_inplace(const Parallel::Communication& comm)
+{
+    this->accumulateRegionSums(comm);
```

Comment:
Could you add a comment here, please, to remind the reader that `accumulateRegionSums()` as a side effect mutates the `GenericOutputModule::initialInplace_` data member?

Reply:
done.